source/includes/inkling2-guide/_next-steps.html.md

```diff
@@ -0,0 +1,17 @@
+# Next Steps
+
+Now that you’ve reviewed this guide, you can:
+
+Review [**Advanced Platform Techniques**][1] (YouTube link) video to help you build and run complex Inkling on the Platform.
```

Comment:
Probably no immediate action needed, but we should flag that these videos have Inkling v1 code in them. At some point, we should record new videos...